.travis.yml

```diff
@@ -8,6 +8,11 @@ env:
   - ARCH=posix CONF=dbg
   - ARCH=posix CONF=opt
   - ARCH=posix CONF=lto
+before_install:
+  - [ X"$TRAVIS_OS_NAME" = X"osx" ] && brew install erlang
```

Comment:
I've used that with ksh where expanding `[ "$x" = something ]`  where x is an empty string resulted in a syntax error, e.g.: http://cvsweb.openbsd.org/cgi-bin/cvsweb/src/etc/rc?rev=1.460&content-type=text/x-cvsweb-markup; however this may be redundant as `$TRAVIS_OS_NAME` is meant to be always present